mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterActor.java

```diff
@@ -192,6 +195,8 @@ public Receive createReceive() {
                 .match(ResourceOverviewRequest.class, this::onResourceOverviewRequest)
                 .match(TaskExecutorInfoRequest.class, this::onTaskExecutorInfoRequest)
                 .match(TaskExecutorGatewayRequest.class, this::onTaskExecutorGatewayRequest)
+                .match(DisableTaskExecutorRequest.class, this::onDisableTaskExecutorRequest)
+                .match(DeleteDisabledTaskExecutorRequest.class, this::onDeleteDisabledTaskExecutorRequest)
```

Comment:
who/how is this action used?

Reply:
Titus Migration logic will call `onDisableTaskExecutorRequest` while the reaper (yet to be implemented) will call `onDeleteDisabledTaskExecutorRequest` after killing the container using spinnaker API.

Reply:
"onDeleteDisabledTaskExecutorRequest": what does this do in RCActor?